segmentation:
  front_crop: 0.03
  back_crop: 0.04
  top_crop: 0.04
  bottom_crop: 0.01
  left_crop: 0.03
  right_crop: 0.03